requirements.txt

```diff
@@ -44,6 +44,7 @@ gcsfs>=2024.5.0
 # adlfs
 
 trl @ git+https://github.com/huggingface/trl.git@31d02cfb795284591a084416b9dcb7bef5d08924
+# trl==0.12.0
```

Comment:
doesn't seem to actually be using `trl==0.12.0`

Reply:
I think I simply forgot to revert my comment post testing, just fixed it. But yeah `trl==0.12.0` is correctly integrated.